internals/daemon/daemon.go

```diff
@@ -717,22 +742,6 @@ func (d *Daemon) doReboot(sigCh chan<- os.Signal, waitTimeout time.Duration) err
 	return fmt.Errorf("expected reboot did not happen")
 }
 
-var shutdownMsg = "reboot scheduled to update the system"
```

Comment:
This has been moved to ```internals/systemd/shutdown```. Note that I am reusing the ```shutdown``` command group name here, where the command offers halt, power-off and reboot functionality.